packages/devtools_app/lib/src/ui/flutter/service_extension_widgets.dart

```diff
@@ -227,16 +230,50 @@ class _RegisteredServiceExtensionButtonState
   Widget build(BuildContext context) {
     if (_hidden) return const SizedBox();
 
-    return FlatButton(
-      onPressed: _disabled ? null : _click,
-      child: Row(children: [
-        getIconWidget(widget.serviceDescription.icon),
-        if (widget.showTitle)
-          Padding(
-            padding: const EdgeInsets.only(left: 8.0),
-            child: Text(widget.serviceDescription.title),
-          ),
-      ]),
+    return InkWell(
+      onTap: _click,
+      child: Container(
+        constraints: const BoxConstraints(minWidth: 48.0, minHeight: 48.0),
```

Comment:
why not width 48, height 48? something will be badly wrong if the icons are more than 48 pixels high

Reply:
It looked like the original intention of this widget was to also show text on the side.

I'm going to rewrite this build method to only show the icon.

Reply:
sg. Feel free to also leave as is.